templates/foundry/solidity/.github/workflows/test.yml

```diff
@@ -0,0 +1,34 @@
+name: test
+
```

Comment:
Is this a workflow to be a part of this template, or do we want to run this workflow in this repo as part of PRs?